### ISSUE TITLE
glutPassiveMotionFunc() happens periodically on windows while the mouse stands still

### ISSUE THREAD

**Issue by corwinn** [FreeGLUTProject/freeglut#136]
Start it from a console and position the mouse pointer over the view-port. Then observe the console output without moving the mouse.
I'm using the "github" "gist" for a 1st time, so I hope you can see the min. code example:
https://gist.github.com/corwinn/784592dcd2d26bb63e1280015a2d1706
.
.
An example build command (it should build just fine at the VS as well, should you happen to have the env.):
`x86_64-w64-mingw32-clang++ -Wall -Wextra -Wshadow -mwindows -mconsole -static -DFREEGLUT_STATIC -O0 -g -std=c++14 freeglut_issue.cpp -o freeglut_issue.exe -lopengl32 -lglu32 -lfreeglut_static -lwinmm`
(dynamic linking yields the same result)

The behavior isn't observed with GLX.

**Comment by dcnieho** [comment]
can you post some example output to your gist as well? If i recall
correctly, freeglut simply forwards window events it receives from Windows.
Also, would you have the opportunity to compile this with MSVC to see if
its something about the headers mingw ships?

**Comment by corwinn** [comment]
Yes - it does - it forwards them: I just reviewed your code at `fg_main_mswin.c`. Its OK. I just finished reading this: ["Why do I get spurious WM_MOUSEMOVE messages?"](https://devblogs.microsoft.com/oldnewthing/20031001-00/?p=42343) after I observed the same behavior at a simple VS-built C# program. "oldnewthing" part of that link is quite in place, in regards to the OS in question :)

Filtering at the handler level is the right thing to do. Not your issue at all.
